src/encoded/tests/test_types_experiment.py

```diff
@@ -12,80 +12,173 @@ def base_experiment(testapp, lab, award, cell_free):
     }
     return testapp.post_json('/experiment', item, status=201).json['@graph'][0]
 
-def test_isogenic_replicate_type(testapp, base_experiment, donor_1, donor_2,biosample_1, biosample_2, library_1, library_2, replicate_1_1, replicate_2_1 ):
+
```

Comment:
For the test refactor, I think you can make `create_fly_experiment` a pytest fixture:
```suggestion
@pytest.fixture
```

Reply:
k